packages/devtools_app/lib/src/flutter/console.dart

```diff
@@ -104,25 +103,23 @@ class _ConsoleOutputState extends State<_ConsoleOutput> {
     final textStyle =
         theme.textTheme.bodyText2.copyWith(fontFamily: 'RobotoMono');
 
-    return OutlineDecoration(
-      child: Scrollbar(
```

Comment:
I'm not a fan of removing the outline from the console, now everywhere that we use it we need to wrap it in the OutlineDecoration component, but I see how this'd be problematic when laying this out with an external title widget.

Reply:
I think the presence of an outline is mostly dictated by the parent container and layout rather than something specific to the widget. For instance, for the debugger views on the left side of the page:

<img width="350" alt="Screen Shot 2020-05-28 at 9 06 03 AM" src="https://user-images.githubusercontent.com/1269969/83165548-85f8dc00-a0c2-11ea-9c4d-79f8d6159905.png">

if the area widgets had outlines around them as part of their definition, they'd have double outlines for the places where they abutted.

I guess I'm ok w/ a use of this widget knowing that it had to wrap it in an Outline for best results -

Reply:
Haha OK, you convinced me. I wanted to have a completely self-contained widget, but that column looks like a good case for "the container controlling the outline".